File: app/src/main/java/com/k4dima/chat/core/api/LemonadeService.java
package com.k4dima.chat.core.api;

import com.k4dima.chat.core.model.ResponseMessages;
import com.google.gson.JsonObject;

import io.reactivex.Observable;
import retrofit2.http.Body;
import retrofit2.http.POST;

public interface LemonadeService {
    @POST("/message")
    Observable<ResponseMessages> message(@Body JsonObject message);
}

File: app/src/main/java/com/k4dima/chat/core/api/LemonadeService.kt
package com.k4dima.chat.core.api

import com.google.gson.JsonObject
import com.k4dima.chat.core.model.ResponseMessages
import io.reactivex.Single
import retrofit2.http.Body
import retrofit2.http.POST

interface LemonadeService {
    @POST("/message")
    fun message(@Body message: JsonObject): Single<ResponseMessages>
}